components/src/dynamo/sglang/request_handlers/multimodal/worker_handler.py

```
@@ -787,42 +833,92 @@ async def _process_prefill_generation(
+                process_result(first_result)
+
+                async for result in results:
+                    process_result(result)
         finally:
+            if first_result_task is not None:
+                if not first_result_task.done():
+                    first_result_task.cancel()
+                try:
+                    await first_result_task
+                except (asyncio.CancelledError, Exception):
+                    pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)